src/util/Options.js

```diff
@@ -69,6 +69,7 @@
  * [User Agent](https://discord.com/developers/docs/reference#user-agent) header
  * @property {PresenceData} [presence={}] Presence data to use upon login
  * @property {IntentsResolvable} intents Intents to enable for this connection
+ * @property {number} [waitGuildTimeout=15000] Time in milliseconds that Clients with the GUILDS intent should wait for missing guilds to be recieved before starting the bot. If not specified, the default is 15 seconds.
```

Comment:
To comply with the line length limit:
```suggestion
 * @property {number} [waitGuildTimeout=15000] Time in milliseconds that Clients with the GUILDS intent should wait for
 * missing guilds to be recieved before starting the bot. If not specified, the default is 15 seconds.
```